ansible/cluster-setup/site.yaml

```diff
@@ -52,6 +52,23 @@
           zone: "{{ cluster_host[1:] }}"
       values_files: ["../../charts/vault/values.yaml"]
       create_namespace: true
+- name: Install Vault Config Chart.
+  hosts: 'localhost'
+  tasks:
+  - name: Create sshKey Secret
+    with_items: "{{ env }}"
+    shell: "printf \"apiVersion: v1\nkind: Secret\nmetadata:\n  name: vault-github-plugin-creds\ntype: kubernetes.io/ssh-auth\nstringData:\n  github_app_id: $(echo \"\\\"\"$GITHUB_APP_ID\"\\\"\")\n  github_org: $(echo \"\\\"\"$GITHUB_ORGANIZATION\"\\\"\")\n  ssh-privatekey: |\n$(cat $GITHUB_KEY_FILE | sed 's/^/    /')\" | oc apply -f - -n vault-admin"
```

Comment:
I feel like there must be a cleaner way to do this.... but I have no idea what that would be offhand. 

Maybe we can put a comment in here just to revisit this in the future if someone has some time to blow?

Reply:
I agree.  I was thinking the same thing but I spent enough time trying to get this working.  I'll add the comment